modules/core/src/main/java/org/apache/ignite/internal/pagemem/store/PageStore.java

```diff
@@ -17,14 +17,20 @@
 
 package org.apache.ignite.internal.pagemem.store;
 
+import java.io.Closeable;
 import java.nio.ByteBuffer;
 import org.apache.ignite.IgniteCheckedException;
 import org.apache.ignite.internal.processors.cache.persistence.StorageException;
 
 /**
  * Persistent store of pages.
  */
-public interface PageStore {
+public interface PageStore extends Closeable {
```

Comment:
It used for PageStore recovery. You are not able to file the `close` method usages since `try-with-resource` is used. Please, check `sendDelta` method of the `LocalSnapshotSender`.